kaldb/src/main/java/com/slack/kaldb/zipkinApi/ZipkinService.java

```diff
@@ -58,7 +76,147 @@ public HttpResponse getTraces(
 
   @Get("/api/v2/trace/{traceId}")
   public HttpResponse getTraceByTraceId(@Param("traceId") String traceId) throws IOException {
-    String output = "[]";
+
+    String queryString = "trace_id:" + traceId;
+    // since the traceById doesn't support a start date param max out to 1 days for now
+    // todo: read this value from the replica config?
+    long startTime =
+        TimeUnit.NANOSECONDS.toMillis(LocalDate.now().minusDays(1).atStartOfDay().getNano());
```

Comment:
> I think we should take startTime and endTime as a param instead of assuming start time as now.

How do you think this will work? The spec ( https://zipkin.io/zipkin-api/#/default/get_trace__traceId_ ) doesn't support more than just `traceId` param. Should we add start and end time and just use defaults?